<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkdownTextEditor</class>
 <widget class="QWidget" name="MarkdownTextEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarkdownTextEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PlainTextEdit" name="plainTextEdit"/>
   </item>
   <item>
    <widget class="QFrame" name="searchFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="replaceToggleButton">
        <property name="toolTip">
         <string>Toggle Replace</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="menu-expanded"/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="4">
         <widget class="QToolButton" name="searchPrevious">
          <property name="toolTip">
           <string>Previous Match</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="go-previous">
            <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="modeLabel">
          <property name="text">
           <string>Mode:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QToolButton" name="searchClose">
          <property name="toolTip">
           <string>Close</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="window-close">
            <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="LineEdit" name="replaceLineEdit">
          <property name="placeholderText">
           <string>Replace</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="replaceLabel">
          <property name="text">
           <string>Replace:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="searchCountLabel">
          <property name="text">
           <string notr="true">-/-</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="searchLabel">
          <property name="text">
           <string>Find:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="LineEdit" name="searchLineEdit">
          <property name="placeholderText">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="modeComboBox">
            <item>
             <property name="text">
              <string>Plain text</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Whole words</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Regular expression</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="replaceButton">
            <property name="toolTip">
             <string>Replace one text occurrence</string>
            </property>
            <property name="text">
             <string>Replace</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="replaceAllButton">
            <property name="toolTip">
             <string>Replace all text occurrences</string>
            </property>
            <property name="text">
             <string>Replace all</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="5">
         <widget class="QToolButton" name="searchNext">
          <property name="toolTip">
           <string>Next Match</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="go-next">
            <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QToolButton" name="searchFirst">
          <property name="toolTip">
           <string>First Match</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="go-first"/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QToolButton" name="searchLast">
          <property name="toolTip">
           <string>Last Match</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="go-last"/>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>gui/ContextMenuOverride.h</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>gui/ContextMenuOverride.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../../../Downloads/qmarkdowntextedit-develop/media.qrc"/>
 </resources>
 <connections/>
</ui>
